### PR TITLE
remove imports v1.1 endpoint

### LINKED ISSUE #4332: remove imports v1.1 endpoint
choose one and promote to "new v1".

Branch `4332-remove-imports-v1_1`

### PR DESCRIPTION
**Pull request by Kartones**
Closes #4332 

Needs removal of `active_record_imports_service_endpoint` and `active_record_import_endpoint`flags

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4870/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4871/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4880/))

**Comment by Kartones** [comment]
@juanignaciosl CR please

File: config/routes.rb [hunk 1]
```diff
@@ -388,18 +388,13 @@
     post '(/user/:user_domain)(/u/:user_domain)/api/v1/uploads' => 'uploads#create', as: :api_v1_uploads_create
 
     # Imports
-    post   '(/user/:user_domain)(/u/:user_domain)/api/v1/imports'                          => 'imports#create',                      as: :api_v1_imports_create
-    get    '(/user/:user_domain)(/u/:user_domain)/api/v1/imports/:id'                      => 'imports#show',                        as: :api_v1_imports_show
     get    '(/user/:user_domain)(/u/:user_domain)/api/v1/imports'                          => 'imports#index',                       as: :api_v1_imports_index
```
Comment (juanignaciosl) [inline config/routes.rb]: Is this ok? :-\ Url alias and path are duplicated.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4882/))

**Comment by juanignaciosl** [comment]
LGTM

**Comment by Kartones** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4886/))

**Pull request merged**